benches/generators.rs

```diff
@@ -72,18 +70,18 @@ gen_uint!(gen_u32_xorshift, u32, XorShiftRng);
 gen_uint!(gen_u32_hc128, u32, Hc128Rng);
 gen_uint!(gen_u32_isaac, u32, IsaacRng);
 gen_uint!(gen_u32_isaac64, u32, Isaac64Rng);
-gen_uint!(gen_u32_chacha, u32, ChaChaRng);
 gen_uint_new!(gen_u32_std, u32, StdRng);
 gen_uint_new!(gen_u32_os, u32, OsRng);
 
 gen_uint!(gen_u64_xorshift, u64, XorShiftRng);
 gen_uint!(gen_u64_hc128, u64, Hc128Rng);
 gen_uint!(gen_u64_isaac, u64, IsaacRng);
 gen_uint!(gen_u64_isaac64, u64, Isaac64Rng);
-gen_uint!(gen_u64_chacha, u64, ChaChaRng);
 gen_uint_new!(gen_u64_std, u64, StdRng);
 gen_uint_new!(gen_u64_os, u64, OsRng);
 
+// Do not test JitterRng like the others by running it RAND_BENCH_N times per,
+// measurement, because it is way to slow. Only run it once
```

Comment:
I think it's time you stopped confusing 'to' and 'too'!

Reply:
You are right. I will try too not make to many mistakes with them anymore. Starting tomorrow 😄.